tests/test_http_downloader.py

```diff
@@ -0,0 +1,145 @@
+import tempfile
+from unittest.mock import patch
+
+import pytest
+
+from datasetinsights.io.downloader.http_downloader import HTTPDownloader
+from datasetinsights.io.exceptions import ChecksumError
+
+
+def test_read_checksum_from_local_file():
+    # arrange
+    with tempfile.NamedTemporaryFile(mode="w+") as tmp:
+        tmp.write("123456")
+        tmp.flush()
+        # act
+        checksum = HTTPDownloader.get_checksum_from_file(tmp.name)
+        # assert
+        assert checksum == 123456
+
+
+@pytest.mark.parametrize("filepath", ["http://some/path", "https://some/path"])
```

Comment:
It will be nice to add  a few test cases when a given `filepath` will raise `ValueError`.

Reply:
The test below does this.

Reply:
Are there any cases you have in mind?

Reply:
A few invalid cases: ["zvsssdfsdg", ":path with space", ""]

Reply:
Added [here](https://github.com/Unity-Technologies/datasetinsights/pull/47/commits/93e30c6571d0fb0db175dacc03e7cfb072fc5a0e)